saas/static/js/djaodjin-saas-angular.js

```diff
@@ -1046,8 +1058,13 @@ metricsControllers.controller("metricsCtrl",
 
             queryset.unit = unit;
             queryset.scale = scale;
-            queryset.data = resp.data.table;
-
+            if($scope.timezone === 'local'){
+                queryset.data = convertDataTimeToLocal(resp.data.table)
```

Comment:
`convertDataTimeToLocal` modifies dates in-place and returns the last `values` array. It does not return `resp.data.table` as expected here. As a result the table in saas/templates/metrics/base.html is not generated correctly.
Lines 1061 to 1067 could be replaced by:

    queryset.data = resp.data.table;
    if($scope.timezone === 'local'){
        convertDataTimeToLocal(queryset.data);
    }